### PR TITLE
[candidate_profile] Add media widget

### PR DESCRIPTION
**Pull request by driusan**
This adds a widget to the candidate dashboard to display
any media associated with the candidate from the media
module. The visualization of how the data is displayed
is the same as the "Document Repository" widget on the
main LORIS dashboard.

### CODE REVIEW

File: modules/media/php/module.class.inc [hunk 1]
```diff
@@ -58,4 +41,47 @@ class Module extends \Module
...
+            $factory   = \NDB_Factory::singleton();
+            $baseurl   = $factory->settings()->getBaseURL();
+            $candidate = $options['candidate'];
+            if ($candidate === null) {
+                return [];
+            }
+            $db = $factory->database();
+
+            $media = $db->pselect(
+                "SELECT file_name as Filename, date_uploaded as UploadDate
```
Comment (johnsaigle) [inline modules/media/php/module.class.inc]:
```suggestion
                "SELECT file_name as Filename, last_modified as LastModified
```
This was changed in https://github.com/aces/Loris/pull/5256

Comment (driusan) [reply modules/media/php/module.class.inc]: done in latest commit, the jsx had to be modified too.

**Comment by johnsaigle** [comment]
Screen Shot 2020-03-09 at 12 32 32

When there is no media, the card should be hidden

**Comment by driusan** [comment]
@johnsaigle done in latest commit

File: modules/media/jsx/CandidateMediaWidget.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+function CandidateMediaWidget(props) {
+    let files = [];
+    for (let i = 0; i < props.Files.length; i++) {
+        const file = props.Files[i];
+        files.push(
+            <a className="list-group-item" key={i}
+                href={props.BaseURL
+                    + '/media/ajax/FileDownload.php?File='
+                    + encodeURIComponent(file.Filename)}>
```
Comment (jesscall) [inline modules/media/jsx/CandidateMediaWidget.js]: you should get the config media path and append it here

Comment (jesscall) [reply modules/media/jsx/CandidateMediaWidget.js]: otherwise i am getting 404 error

Comment (driusan) [reply modules/media/jsx/CandidateMediaWidget.js]: Are you sure? This is the same way it's done in mediaIndex.js:

```
        const downloadURL = loris.BaseURL + '/media/ajax/FileDownload.php?File=' +
          encodeURIComponent(row['File Name']);
```

Comment (jesscall) [reply modules/media/jsx/CandidateMediaWidget.js]: you're right. might have something to do with how i upgraded. ill investigate and let you know if it's actually an issue with the PR

**Pull request merged**